test/jobs/close_patients_job_test.rb

```diff
@@ -134,7 +135,7 @@ def teardown
     assert_equal(closed_email.header['subject'].value, 'Sara Alert Reporting Complete')
     assert_includes(
       closed_email.text_part.body.to_s.gsub("\r", ' ').gsub("\n", ' '),
-      "Sara Alert monitoring for #{patient.initials_age('-')} completed on #{DateTime.now.strftime('%m-%d-%Y')}! Thank you for your participation."
+      "Sara Alert monitoring for #{patient.initials_age('-')} completed on #{DateTime.now.utc.strftime('%m-%d-%Y')}! Thank you for your participation."
```

Comment:
Related lines:
* https://github.com/SaraAlert/SaraAlert/blob/master/app/mailers/patient_mailer.rb#L199
* https://github.com/SaraAlert/SaraAlert/blob/master/app/mailers/patient_mailer.rb#L222

Does this make sense to be in UTC since we are sending this to the monitoree?

cc: @holmesie @tstrass

Reply:
If server time is UTC then this has always been in UTC. IMO it would make more sense for this to be in patient local time but I also think that is out of scope for this PR

Reply:
https://tracker.codev.mitre.org/browse/SARAALERT-1563